SimTracker/SiPhase2Digitizer/interface/Phase2TrackerMonitorDigi.h

```diff
@@ -0,0 +1,52 @@
+#ifndef Phase2TrackerMonitorDigi_h
+#define Phase2TrackerMonitorDigi_h
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+
+// DQM Histograming
+class DQMStore;
+class MonitorElement;
+class PixelDigi;
+class Phase2TrackerDigi;
+
+class Phase2TrackerMonitorDigi : public edm::EDAnalyzer {
```

Comment:
@suchandradutta - it should be a thread-safe one:
```
edm::one::EDAnalyzer<edm::one::WatchRuns>
```
Please, see:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedAnalysisEDAnalyzer